doc/source/user_guide/offline/file_session.rst

```diff
@@ -0,0 +1,208 @@
+   0.001690600193527586
+
+
+Multi phase
+-----------
+
+.. code-block:: python
+
+  >>> from ansys.fluent.core import examples
+  >>> from ansys.fluent.core.file_session import FileSession
+
+  >>> case_file_name = examples.download_file("mixing_elbow_mul_ph.cas.h5", "pyfluent/file_session")
+  >>> data_file_name = examples.download_file("mixing_elbow_mul_ph.dat.h5", "pyfluent/file_session")
+  >>> file_session = FileSession()
+  >>> file_session.read_case(case_file_name)
+  >>> file_session.read_data(data_file_name)
+
+  >>> file_session.fields.field_info.get_scalar_field_range("phase-2:SV_P")
+  [0.0, 1.5435200335871788e-11]
+  >>> file_session.fields.field_info.get_scalar_fields_info()
+  {'phase-1:SV_ARTIFICIAL_WALL_FLAG': {'display_name': 'SV_ARTIFICIAL_WALL_FLAG',
+   'section': 'field-data',
+   'domain': 'phase-1'},
+   'phase-1:SV_DENSITY': {'display_name': 'SV_DENSITY',
+   'section': 'field-data',
+   'domain': 'phase-1'},
+   .....
+   'phase-4:': {'display_name': '',
+   'section': 'field-data',
+   'domain': 'phase-4'}}
+   >>> file_session.fields.field_info.get_vector_fields_info()
+   {'phase-1:velocity': {'x-component': 'phase-1: SV_U',
+    'y-component': 'phase-1: SV_V',
+    'z-component': 'phase-1: SV_W'},
+    .....
+    'phase-4:velocity': {'x-component': 'phase-4: SV_U',
+    'y-component': 'phase-4: SV_V',
+    'z-component': 'phase-4: SV_W'}}
+   >>> transaction = file_session.fields.field_data.new_transaction()
+   >>> transaction.add_scalar_fields_request("phase-1:SV_DENSITY", [30])
+   >>> transaction.add_vector_fields_request("phase-1:velocity", [30])
+   >>> transaction.get_fields()
+   {(('type', 'scalar-field'),
+     ('dataLocation', 1),
+     ('boundaryValues',
+      False)): {30: {'phase-1:SV_DENSITY': array([1.225, .....          1.225])}},
+    (('type',
+      'vector-field'),): {30: {'phase-1:velocity': array([0., ..... 0.]),
+      'vector-scale': array([0.1])}}}
+   >>> from ansys.fluent.core.services.field_data import SurfaceDataType
+   >>> file_session.fields.field_data.get_surface_data(SurfaceDataType.Vertices, [30])[30].size
+   79
+   >>> ffile_session.fields.field_data.get_surface_data(SurfaceDataType.Vertices, [30])[30][50].x
+   0.14896461503555408
+   >>> file_session.fields.field_data.get_scalar_field_data("phase-1:SV_P", surface_name="wall-elbow").size
+   2168
+   >>> file_session.fields.field_data.get_scalar_field_data("phase-1:SV_P", surface_name="wall-elbow")[1100].scalar_data
+   1.4444035696104466e-11
+   >>> file_session.fields.field_data.get_vector_field_data("phase-2:velocity", surface_name="wall-elbow").size
+   2168
+   >>> file_session.fields.field_data.get_vector_field_data("phase-2:velocity", surface_name="wall-elbow")[1000].x
+   0.0
+
+
+Sample usage
+------------
+
+You can use the Graphics toolbar to display the mesh, contour and vector data respectively.
+
+
+.. code-block:: python
+
+  >>> from ansys.fluent.visualization import set_config
```

Comment:
Should the visualization code be moved to the visualization repo?

Reply:
Yeah, that sounds good. In the description at line 165, we can say `... use the ansys-fluent-visualization package ...`?  Not sure what is the "Graphics toolbar" mentioned in that line.

Reply:
Yes, it looks like that text was probably copied from somewhere else. I'll keep it relevant.